.github/workflows/test.yml

```diff
@@ -0,0 +1,180 @@
+name: test
+
+on:
+  push:
+    branches:
+      - "ghactions-test" #remove when merged
+      - 'master'
+    tags:
+      - 'v*'
+      - 'dockerfile/*'
+  pull_request:
+    branches:
+      - 'master'
+
+env:
+  REPO_SLUG_ORIGIN: "moby/buildkit:latest"
+
+jobs:
+  test-base:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Cache integration-tests
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildkit-cache/integration-tests
+          key: ${{ runner.os }}-buildkit-integration-tests-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildkit-integration-tests-
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          driver-opts: image=${{ env.REPO_SLUG_ORIGIN }}
+      -
+        name: Build integration-tests
+        run: |
+          ./hack/build_ci_first_pass integration-tests
+        env:
+          CACHE_DIR: /tmp/.buildkit-cache/integration-tests
+      -
+        name: Dump context
+        if: always()
+        uses: crazy-max/ghaction-dump-context@v1
+
+  test:
+    runs-on: ubuntu-latest
+    needs: [ test-base ]
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          -
+            pkg: ./client
+            flags: -v --timeout=20m
+            skip-it: 0
```

Comment:
`skip-it` is a confusing name for new reader. `integration-tests` is better. does `skip-it: 0` need to be defined or isn't it default.